cpp/daal/src/algorithms/dtrees/gbt/regression/gbt_regression_predict_dense_default_batch_impl.i

```diff
@@ -61,6 +60,7 @@ class PredictRegressionTask
 {
 public:
     typedef gbt::internal::GbtDecisionTree TreeType;
+    typedef gbt::prediction::internal::TileDimensions<algorithmFPType> DimType;
```

Comment:
Can you please look at the approach to block size calculation used in K-means: https://github.com/oneapi-src/oneDAL/blob/master/cpp/daal/src/algorithms/kmeans/kmeans_lloyd_batch_impl.i#L140

DAAL_SAFE_CPU_CALL macro sets up default block size for third-party CPUs.

Reply:
done